server.py

```diff
@@ -41,8 +40,6 @@
 
 @asyncio.coroutine
 async def init(address, port, services, users):
-    context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
-    context.load_cert_chain(SSL_CERT_FILE, SSL_KEY_FILE)
```

Comment:
I don't think we need the SSL_CERT_FILE, SSL_KEY_FILE variables added at the top of this file. The auth_svc is the only thing using it, which looks like we can strip it out of there as well.

Reply:
Unfortunately we do need those variables - they get added to the Auth_Svc and passed to the Adversary Mode plugin to render the `conf.yml`.  I can move the cert to adversary mode if you'd prefer?

Reply:
ooh yeah. let's do that.